projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.service.ts

```diff
@@ -52,6 +67,27 @@ export class OrderDetailsService {
       }),
       shareReplay({ bufferSize: 1, refCount: true })
     );
+
+    this.globalMessageService
+      .get()
+      .pipe(filter(messageEntities => !!Object.keys(messageEntities).length))
+      .subscribe(messageEntities => {
+        const messages =
+          messageEntities && messageEntities[GlobalMessageType.MSG_TYPE_ERROR];
+        if (
+          messages &&
+          messages.some(
+            message =>
+              message.raw ===
+              'OrderModel with guid ' +
```

Comment:
This is looking for an exact match? Because basically I don't think it will ever really say `0` months. Is it the only way to know an order is expired? If so maybe add some regex in here for the number of months.